src/psyclone/tests/utilities.py

```diff
@@ -470,8 +470,7 @@ def get_base_path(api):
     api_2_path = {"dynamo0.1": "dynamo0p1",
                   "dynamo0.3": "dynamo0p3",
                   "nemo": "../nemo/test_files",
-                  "gocean1.0": "gocean1p0",
-                  "gocean0.1": "gocean0p1"}
+                  "gocean1.0": "gocean1p0"}
```

Comment:
Copyright date pls.

Reply:
Again the merge solved this, maybe I should have waited in the first place :)